swagger.yaml

```diff
@@ -4228,6 +4232,12 @@ definitions:
         type: string
         x-nullable: true
         enum: ['auto-assign', 'master', 'worker']
+      host_labels:
```

Comment:
```suggestion
      node_labels:
```

Reply:
The other properties are named host_role, host_name etc. Do you still want me to change it to node_labels?

Reply:
That's a good point.  We usually refer to the machine itself as "host", and the openshift/kubernetes resource as "node". 
 So host_name makes sense.  I would probably change host_role to "role" or "node_role", but I wouldn't break the API for it.  Since these labels will be applied to the kubernetes node resource, I think node_labels makes the most sense.

Reply:
Understood